rules/windows/credential_access_mimikatz_powershell_module.toml

```diff
@@ -21,39 +21,44 @@ note = """## Triage and analysis
 typically represents an active adversary.
-- Any kind of available host-based events or logs such as Windows Security Events, PowerShell logging and EDR events should
-be used to seek further understanding around the events that led up to the rule as well as activity found shortly after the event.
-- Further examination should include reviewing network logs to determine potential lateral movement.
-- Validate that the source of the Mimikatz activity was not from an authorized source such as automated testing such as
-Atomic Red Team or through offensive/compromise assessments.
-
-### False Positive Analysis
-- This rule should be on the higher confidence side of true positive activity therefore any testing such as offensive
-/compromise engagements should be ruled out before invoking incident response procedures
-
-### Related Rules
-- Mimikatz Memssp Log File Detected
-- Creation or Modification of Domain Backup DPAPI private key
-- Modification of WDigest Security Provider
-
-### Response and Remediation
-- Take immediate action to review, investigate and potentially isolate activity to prevent further post-compromise
-behavior
-- During credential dump compromises, investigate the registry in order to check the number of cached users that have
-used the machine. These users should have their password reset.
+
+More information about Mimikatz components and how to detect/prevent them can be found on [ADSecurity](https://adsecurity.org/?page_id=1821).
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree).
+- Contact the account owner and confirm whether they are aware of this activity.
+- Examine PowerShell, Windows, and endpoint detection and response (EDR) logs to understand what was executed in the host.
+- Further examination should include reviewing network logs to identify potential lateral movement.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Use process name, command line, and file hash to search for occurrences on other hosts.
+- Scope potentially compromised accounts. Analysts can do this by searching for login events (for example, 4624) to the
+target host.
+
+### False positive analysis
+
+- This activity is unlikely to happen legitimately. Benign true positives (B-TPs) can be added as exceptions if necessary.
+
+### Related rules
+
+- Mimikatz Memssp Log File Detected - ebb200e8-adf0-43f8-a0bb-4ee5b5d852c6
+- Creation or Modification of Domain Backup DPAPI private key - b83a7e96-2eb3-4edf-8346-427b6858d3bd
+- Modification of WDigest Security Provider - d703a5af-d5b0-43bd-8ddb-7a5d500b7da5
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- Reset passwords for the user account and other potentially compromised accounts (email, services, CRMs, etc.).
```

Comment:
What are CRMs?

Reply:
A class of systems used by sales teams and ops, like Salesforce, adobe, servicenow, zendesk, etc